stock_available_to_promise_release/tests/common.py

```diff
@@ -55,7 +55,7 @@ def _create_picking_chain(cls, wh, products=None, date=None, move_type="direct")
         if products is None:
             products = []
 
-        group = cls.env["procurement.group"].create(
+        cls.group = cls.env["procurement.group"].create(
```

Comment:
Not sure having this assignation at the class level inside a method which can be called on demand is a good thing.